autoPyTorch/pipeline/base_pipeline.py

```diff
@@ -302,9 +310,11 @@ def _get_base_search_space(
             # if the node isn't a choice we can add it immediately because it
             #  must be active (if it wasn't, np.sum(matches) would be zero
             if not is_choice:
+                # for mypy
+                assert not isinstance(node, autoPyTorchChoice)
```

Comment:
Is this really needed? no need to change, but I wonder why you need this because is_choice is the same as this statement.

Reply:
It assumes the node is autopytorch choice and then gives an arg-type error